packages/wpcom-proxy-request/src/index.js

```diff
@@ -98,13 +98,13 @@ debug( 'using "origin": %o', origin );
  * takes care of WordPress.com user authentication (via the currently
  * logged-in user's cookies).
  *
- * @param {Object} originalParams - request parameters
+ * @param {object} originalParams - request parameters
  * @param {Function} [fn] - callback response
- * @return {XMLHttpRequest} XMLHttpRequest instance
- * @api public
+ * @returns {window.XMLHttpRequest} XMLHttpRequest instance
+ * @public
```

Comment:
In an ES module that exports the public functions I'd vote to remove all these `@public`/`@private` tags.

```suggestion
```

Reply:
Done!